### ISSUE TITLE
test-helpers explanation

### ISSUE THREAD

**Issue by ghost** [oliverjam/learn-unit-testing#5]
Clarification - test-helper.js file not mentioned in readme. Might be good to add some comments about it

**Comment by oliverjam** [comment]
Good point, thanks for highlighting. It's mentioned in the `index.html` where we include the JS files, but I guess not everyone looks in there.

https://github.com/oliverjam/learn-unit-testing/blob/30bf35c4f409dcde2e6206018015d57eeca091ab/workshop/index.html#L8-L16

We can put a bit in the README mentioning that the testing code they wrote in the previous workshop has been included for them.
